### ISSUE TITLE
Multi-task BERT

### ISSUE THREAD

**Issue by JohnGiorgi** [BaderLab/saber#139]
Re-write the BERT model such that is can train on an arbitrary number of datasets, like the `MultiTaskLSTMCRF` model. A few things to keep in mind:

- For each dataset, the model should have its own output layer, optimizer compute its own loss function
- The interface should work the same as it does with the `MultiTaskLSTMCRF`, e.g.

```python
from saber import Saber

saber = Saber()
saber.load_dataset(['dataset_1', 'dataset_2'])

saber.build(model_name='bert-ner')
saber.train()
```

would load two datasets and create a multi-task model for training.
